tests/instancetype_test.go

```diff
@@ -355,6 +356,15 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 				Name: preference.Name,
 				Kind: instancetypeapi.SingularPreferenceResourceName,
 			}
+			vm.Spec.Template.Spec.Domain.Devices.Interfaces = append(vm.Spec.Template.Spec.Domain.Devices.Interfaces, v1.Interface{
+				Name: "default",
+			})
+			vm.Spec.Template.Spec.Networks = append(vm.Spec.Template.Spec.Networks, v1.Network{
```

Comment:
Will it also work with the default networks if no other networks are specified?

Reply:
We need to define both the interface and the network. If either one is missing, it will result in an error. The default values are being set only if there is no interface and no network at the same time and the interface has the `InterfaceBindingMethod` already set.

Reply:
ACK